server/chat.ts

```diff
@@ -1647,6 +1652,18 @@ export const Chat = new class {
 			entry.hasRoomPermissions = /\bthis\.can\([^,)\n]*, [^,)\n]*,/.test(handlerCode);
 			entry.broadcastable = /\bthis\.(?:canBroadcast|runBroadcast)\(/.test(handlerCode);
 
+			// assign SOME properties from the base command if the current command uses CommandContext.run.
+			const runsCommand = /this.run\((?:'|"|`)(.*?)(?:'|"|`)\)/.exec(handlerCode);
+			if (runsCommand) {
+				const [, , baseCommand] = runsCommand;
```

Comment:
Have you tested this? Shouldn't this need to be one comma and not two?

Reply:
it's two, since the check for " or ', etc also returns a value in exec

Reply:
No it doesn't. That's what the `?:` part prevents. You _definitely_ didn't test this because I tested it just now.

Reply:
```
> [.../this.run\((?:'|"|`)(.*?)(?:'|"|`)\)/.exec(`this.run('test')`)]
["this.run('test')", "test"]
```

Reply:
oh, okay then - my bad, sorry